tests/wicked/startandstop_sut.pm

```diff
@@ -82,6 +82,33 @@ sub run {
     $results{3} = $res ? "PASSED" : "FAILED";
     mutex_create("test_3_ready");
 
+    $self->before_scenario('Test 8', 'Bridge - ifdown, remove one config, ifreload, ifdown, ifup', $iface);
+    $self->get_from_data('wicked/ifcfg/br0',    $config);
+    $self->get_from_data('wicked/ifcfg/dummy0', $dummy);
+    $self->setup_bridge($config, $dummy, 'ifup');
+    assert_script_run("wicked ifdown --timeout infinite br0");
+    assert_script_run("wicked ifdown --timeout infinite dummy0");
+    $res = script_run("ip link|grep 'dummy0\|br0'");
+    if ($res == 0) {
+        $results{8} = "FAILED";
+    }
+    else {
+        assert_script_run("rm $config");
+        assert_script_run("wicked ifreload --timeout infinite all");
+        my $res1 = script_run("ip link|grep br0");
+        my $res2 = script_run("ip link|grep dummy0");
+        my $res3 = script_run("ip link|grep eth0");
+        if (!$res1 || $res2 || $res3) {
+            $results{8} = "FAILED";
+        }
+        else {
+            assert_script_run("wicked ifdown --timeout infinite all");
+            assert_script_run("wicked ifup --timeout infinite all");
+            $results{8} = $self->get_test_result("host", "");
```

Comment:
No need for passing empty string as ip_version since https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/1dedd06caaeefbf529732d20989956e2f82227f7#diff-7c6669e0bb0ed1f0eed4a62dc81f200dR93